lib/service/database.js

```diff
@@ -87,7 +87,9 @@ DatabaseService.prototype.getMetadataFromAffectedTables = function(node, skip, c
         var defaultLastUpdatedTime = new Date('1970-01-01T00:00:00.000Z');
         var affectedTables = [];
         var lastUpdatedTimes = [];
-        var rows = resultSet.rows || [defaultLastUpdatedTime, null, null];
+        var rows = (Array.isArray(resultSet.rows) && resultSet.rows.length > 0) ?
+            resultSet.rows :
+            [defaultLastUpdatedTime, null, null];
```

Comment:
If we do that we don't enter in the [rows.forEach](https://github.com/CartoDB/camshaft/pull/282/files#diff-b62008d801fcb1406bdf7ff5e30a7a9bR93) which leads to had an empty array for `lastUpdatedTimes` and therefore we have an `Invalid Date` when we try to get the Max date as we have before